db/migrate/20180719162710_add_owner_and_group_to_auth.rb

```diff
@@ -0,0 +1,35 @@
+class AddOwnerAndGroupToAuth < ActiveRecord::Migration[5.0]
+  class Authentication < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+
+    include ReservedMigrationMixin
+    include MigrationStubHelper
+  end
+
+  def up
+    add_column :authentications, :evm_owner_id, :bigint
+    add_column :authentications, :miq_group_id, :bigint
+    add_column :authentications, :tenant_id, :bigint
```

Comment:
@gtanzillo @lpichler hm, I think that for the rest of the cloud objects, we assign ```cloud_tenant_id```, then the rbac goes through Tenant -> mapped CloudTenant -> cloud objects, do I remember it right?

Reply:
@agrare Sort of... Once the `CloudTenant` is mapped to a ManageIQ `Tenant` RBAC will use that for tenancy.